scripts/dracula.sh

```diff
@@ -2,28 +2,14 @@
 # setting the locale, some users have issues with different locales, this forces the correct one
 export LC_ALL=en_US.UTF-8
 
-get_tmux_option() {
-  local option=$1
-  local default_value=$2
-  local option_value=$(tmux show-option -gqv "$option")
-  if [ -z $option_value ]; then
-    echo $default_value
-  else
-    echo $option_value
-  fi
-}
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $current_dir/utils.sh
 
 main()
 {
   datafile=/tmp/.dracula-tmux-data
 
-  # set current directory variable
-  current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
   # set configuration option variables
-  show_battery=$(get_tmux_option "@dracula-show-battery" true)
-  show_network=$(get_tmux_option "@dracula-show-network" true)
-  show_weather=$(get_tmux_option "@dracula-show-weather" true)
```

Comment:
The option is being removed, why? Its needed later on in the file.

Reply:
I removed those because the plugin scripts will be executed only if presents in `@dracula-plugins` config. But I see that `weather` is a bit tricky because of lines 118-120